library/src/main/res/values/dimens.xml

```diff
@@ -50,6 +50,12 @@
     <dimen name="stream_attach_file_size_text">12sp</dimen>
     <!--Thread-->
     <dimen name="stream_thread_toolbar_radius">10dp</dimen>
+    <!--Date Separator-->
+    <dimen name="stream_date_separator_margin_top">20dp</dimen>
+    <dimen name="stream_date_separator_line_height">1px</dimen>
```

Comment:
Should be dp, not pixel. One pixel on high density displays is not visible anymore.